app/models/ingest_measurement.rb

```diff
@@ -12,14 +12,14 @@ def data_for(query)
       search(query: query).results.map(&:_source)
     end
 
-    def query_sensor_data(device_urn) # rubocop:disable Metrics/MethodLength
+    def query_sensor_data(device_urn,time_range) # rubocop:disable Metrics/MethodLength
```

Comment:
Maybe use default value like

```suggestion
    def query_sensor_data(device_urn, time_range = 'now-8w') # rubocop:disable Metrics/MethodLength
```

so that you won't have to change line 14 of `app/views/device_stories/show.html.erb`.

Reply:
Thanks!